source/includes/usage-examples/code-snippets/deleteOne.java

```diff
@@ -0,0 +1,39 @@
+package usage.examples;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.result.DeleteResult;
+
+public class DeleteOne {
+
+    public static void main(String[] args) {
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+            MongoCollection<Document> collection = database.getCollection("movies");
+
+            Bson query = eq("title", "The Garbage Pail Kids Movie");
+
+            System.out.println(collection.find(query).first());
```

Comment:
Maybe add some descriptive text to this `println`? e.g. "Document to delete: { ... }"

Reply:
Whoops, this was for debugging, will remove.